tests/post/features/service_configuration.feature

```diff
@@ -0,0 +1,10 @@
+@post @local @ci @persistence
+Feature: Cluster and Services Configurations
+    Scenario: Propagation of Service Configurations to underlying Services
+        Given the Kubernetes API is available
+        And pods with label 'app.kubernetes.io/name=dex' are 'Ready'
+        And we have 2 running pod labeled 'app.kubernetes.io/name=dex' in namespace 'metalk8s-auth'
+        And we have a 'metalk8s-dex-config' CSC in namespace 'metalk8s-auth' with '['spec']['deployment']['replicas']' equal to '2'
+        When we update 'metalk8s-dex-config' CSC in namespace 'metalk8s-auth' '['spec']['deployment']['replicas']' to '3'
```

Comment:
I really prefer you use jsonpath syntax here (pretty easy in python you just have to split on `.` and get)

Reply:
You also have those `salt.utils` methods: [`salt.utils.data.traverse_dict`](https://github.com/saltstack/salt/blob/2018.3/salt/utils/data.py#L533) and [`salt.utils.data.traverse_dict_and_list`](https://github.com/saltstack/salt/blob/2018.3/salt/utils/data.py#L551)